business-central-parent/business-monitoring-webapp/src/main/resources/META-INF/services/org.uberfire.java.nio.file.spi.FileSystemProvider

```diff
@@ -15,6 +15,6 @@
 #
 
 org.uberfire.java.nio.fs.jgit.JGitFileSystemProvider  # file system provider, also default (1st)
-# org.uberfire.java.nio.fs.k8s.K8SFileSystemProvider    # can be set as default at runtime by system property: org.kie.workbench.controller.openshift.enabled
+org.uberfire.java.nio.fs.k8s.K8SFileSystemProvider    # can be set as default at runtime by system property: org.kie.workbench.controller.openshift.enabled
```

Comment:
@rhtevan can you confirm this is ok?

Reply:
@adrielparedes It is fine to 'enable' k8s implementation now.

Reply:
@adrielparedes  the changes look good to me.